manifests/q/qarmin/czkawka/cli/5.1.0/qarmin.czkawka.cli.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: qarmin.czkawka.cli
+PackageVersion: 5.1.0
+InstallerType: exe

```

```suggestion
InstallerType: portable
```

Hi @TejasMate,

This package appears to be of `portable` type, but has been submitted a couple of times as `exe`. Can future manifests for this package have the `portable` InstallerType instead of `exe`?